src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -106,7 +106,6 @@ public int SelectedIndex
                     selectedString = GetSelectedStringFromItem(Items.ElementAt(value));
                 }
 
-                OnNodeModified();
```

Comment:
I believe the fix for one of those two issues in 3.0 caused an issue in Revit ( REVIT-217658 ) where a custom dropdown node of an internal-package for DynamoRevit that needed to update on idle was always one update delayed.

We might want to consider the risks/benefits of this change, and if we will backport this change into 2.19 then in Revit we must also integrate the changes for REVIT-217658. We should also test around nodes that have UI updates to make sure there aren't other issues not yet found.